src/pages/Download.js

```diff
@@ -33,299 +33,397 @@ const Download = () => {
-                        <a
-                          href='https://mega.nz/folder/RsFSSZjA#rTNA_4nBbLp0BSTX9-DqSA'
-                          target={'_blank'}
-                          rel='noreferrer'>
-                          (MEGA)
-                        </a>
-                      </li>
-                      <li>
-                        Download old UPBGE versions from&nbsp;
-                        <a
-                          href='https://github.com/UPBGE/upbge/releases'
-                          target={'_blank'}
-                          rel='noreferrer'>
-                          here
-                        </a>&nbsp;
-                        or&nbsp;
-                        <a
-                          href='https://mega.nz/folder/k9MW1KiZ#UOKzjh3IQ0GEgjQ6GUc7ug'
-                          target={'_blank'}
-                          rel='noreferrer'>
-                          here
-                        </a>
-                      </li>
-                    </BulletListInside>
-                  </div>
-                </TabSection>
-              </TabPanel>
-              <TabPanel>
-                <TabSection>
-                  <div>
-                    <SectionTitle>Experimental Builds</SectionTitle>
-                    <Button
-                      className={'mt-2'}
-                      href={
-                        'https://mega.nz/folder/t9EEFSaS#JPiOPSInCZyU-SW_-rhEOQ'
-                      }
-                      target={'_blank'}>
-                      Browse or download
-                    </Button>
-                  </div>
-                  <div>
-                    <p>NOTE:</p>
-                    <BulletListInside>
-                      <li>The repository is hosted inside a MEGA folder</li>
-                    </BulletListInside>
-                  </div>
-                </TabSection>
-                <br />
-                <TabSection>
-                  <div>
-                    <SectionTitle>Requirements</SectionTitle>
-                    <BulletListInside>
-                      <li>CPU: 64-bit, Dual-Core 2GHz CPU with SSE2 support</li>
-                      <li>RAM: minimum 2GB</li>
-                      <li>Display: 1280×768</li>
-                      <li>GPU: minimum 1GB VRAM; OpenGL 3.3+</li>
-                      <li>Hard Drive: minimum 500MB free space</li>
-                    </BulletListInside>
-                  </div>
-                </TabSection>
+                <Tabs forceRenderTabPanel>
+                  <TabSection>
+                    <TabList>
+                      <Tab>Windows</Tab>
+                      <Tab>Linux</Tab>
+                      <Tab>macOS</Tab>
+                      <Tab>Experimental</Tab>
+                    </TabList>
+                    <TabPanel>
+                      <TabSection>
+                        <div>
+                          <SectionTitle>Windows 8.1, 10, & 11</SectionTitle>
```

Comment:
To modify title adding Windows 7 too.

Reply:
Could you explain a bit more? If we include Windows 7 with the rest of the version titles we'll have to state different download titles for the button too.

![image](https://user-images.githubusercontent.com/69180012/155202718-4b351faa-dba5-4e34-bcff-c33b2064d2c1.png)

Reply:
The comment was referring to UPBGE 0.2.x tab + Windows. 
![image](https://user-images.githubusercontent.com/1015605/155208207-abad0965-30ed-4f29-bb16-9084209f744d.png)
Maybe I referred a wrong line. Anyway, UPBGE 0.2.X Windows build works for Windows 7, 8.1, 10 & 11.

Reply:
Ohh, I see now. I thought you were referring to the UPBGE 0.3.x section, not UPBGE 0.2.x section.
Yes, I will fix this. Thanks for pointing that out!